vm-image-spec.yaml

```diff
@@ -272,6 +272,19 @@ files:
               else (pg_current_wal_lsn() - '0/0')::FLOAT8
             end as lsn;
 
+      - metric_name: receive_lsn
```

Comment:
We didn't do it in the past, but I think we need to start doing it, at least right now -- prefix all new compute metrics with `compute_`

Reply:
Fine with me. I'll do it pending the other conversation

Reply:
LGTM, keeping this renaming in mind

I'd also add `compute_` to `current_lsn`. I don't think it's used anywhere, so renaming will be good for consistency